download_dependencies.ps1

```diff
@@ -1,5 +1,8 @@
 Param([parameter(Mandatory=$true,Position=0)][String]$GLFW_VERSION)
 
+# The built .so file will end in .so.3.3 for a version like 3.3.7, to get the correct file we need to pass "3.3" Rename-Item for wayland
+[String]$GLFW_SHORT_VERSION = $GLFW_VERSION.Substring(0, $GLFW_VERSION.LastIndexOf("."))
+
```

Comment:
I would rather we `$GLFW_SHORT_VERSION` as a parameter to this script, as `make_nuget.ps1` already has this value calculated.